src/observable/FromEventObservable.ts

```diff
@@ -78,28 +84,31 @@ export class FromEventObservable<T, R> extends Observable<T> {
    */
   static create<T>(target: EventTargetLike,
                    eventName: string,
-                   selector?: (...args: Array<any>) => T): Observable<T> {
-    return new FromEventObservable(target, eventName, selector);
+                   selector?: (...args: Array<any>) => T,
+                   options?: EventListenerOptions): Observable<T> {
+    return new FromEventObservable(target, eventName, selector, options);
```

Comment:
Personally, I am not a big fan of polymorphic methods in javascript but I can see the point in this case.

Are you thinking of something like this:
```javascript
type SelectorMetodSignature<T> = (...args: Array<any>) => T;

...

create<T>(target: EventTargetLike,
          eventName: string,
          selector?: SelectorMetodSignature<T> | EventListenerOptions,
          options?: EventListenerOptions): Observable<T> {
  if (typeof selector !== 'function') {
    selector = undefined;
    options = selector;
  }
  return new FromEventObservable(target, eventName, <SelectorMetodSignature<T>>selector, options);
}
```